documentation/modules/tracing/proc-configuring-tracers-kafka-clients.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assemblies:
+//
+// assembly-distributed tracing.adoc
+
+[id='proc-configuring-tracers-kafka-clients-{context}']
+= Initializing tracing for Kafka clients
+
+[role="_abstract"]
+Initialize a tracer, then instrument your client applications for distributed tracing.
+You can instrument Kafka producer and consumer clients, and Kafka Streams API applications.
+You can initialize a tracer for OpenTracing or OpenTelemetry.
+
+Configure and initialize a Jaeger tracer using a set of xref:ref-tracing-environment-variables-{context}[tracing environment variables].
+
+.Procedure
+
+In each client application add the dependencies for the tracer:
+
+. Add the Maven dependencies to the `pom.xml` file for the client application:
++
+.Dependencies for OpenTelemetry
+[source,xml,subs="attributes+"]
+----
+<dependency>
+    <groupId>io.opentelemetry</groupId>
+    <artifactId>opentelemetry-sdk-extension-autoconfigure</artifactId>
+    <version>{OpenTelemetryAlphaVersion}</version>
+</dependency>
+<dependency>
+  <groupId>io.opentelemetry.instrumentation</groupId>
+  <artifactId>opentelemetry-kafka-clients-{OpenTelemetryKafkaClient}</artifactId>
+  <version>{OpenTelemetryAlphaVersion}</version>
+</dependency>
+<dependency>
+  <groupId>io.opentelemetry</groupId>
+  <artifactId>opentelemetry-exporter-jaeger</artifactId>
+  <version>{OpenTelemetryVersion}</version>
+</dependency>
+----
++
+.Dependencies for OpenTracing
+[source,xml,subs="attributes+"]
+----
+<dependency>
+    <groupId>io.jaegertracing</groupId>
+    <artifactId>jaeger-client</artifactId>
+    <version>1.3.2</version>
+</dependency>
+<dependency>
+  <groupId>io.opentracing.contrib</groupId>
+  <artifactId>opentracing-kafka-client</artifactId>
+  <version>{OpenTracingKafkaClient}</version>
+</dependency>
+----
+
+. Define the configuration of the tracer using the xref:ref-tracing-environment-variables-{context}[tracing environment variables].
+
+. Create the tracer from the environment variables that you defined in step two:
```

Comment:
I don't get this sentence "from the environment variables" .... I would explain that you are using the OpenTracing or OpenTelemetry API to get a tracer which is initialized accordingly through the environment variables.